src/components/Common/MainMenu.vue

```diff
@@ -96,7 +96,7 @@ export default {
     },
     currentUserName() {
       if (!this.user) {
-        return 'Not authentified'
+        return 'Not authenticated'
```

Comment:
Why it's not `Anonymous` instead ?

Reply:
It is different
```
      if (!this.user) {
        return 'Not authenticated'
      }
      if (this.user.id === -1) {
        return 'Anonymous'
      }
```

Reply:
Actually for Kuzzle it's not since when a request is made without authentication Kuzzle automatically assign the anonymous user.
But maybe in the AC it's different so I will let you judge which way is better on this one

Reply:
Actually, the AC has a slightly different state machine:
* Not authenticated (`currentUser === null`) means the end user has not yet decided whether to start the current session as an authenticated user, or the Anonymous user 👉 we display the login dialog.
* Anonymous (`currentUser === 'Anonymous'`) means the user has decided to start the session as the Anonymous user, thus uses the app with the rights attached to this user.
* Another user (`currentUser === 'luca'`) means the user has decided to login as an authenticated user, which ends up being the same case as the previous one.